jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/Menu.java

```diff
@@ -20,6 +22,16 @@ public String hasPosition() {
         return String.format("top: %s; left: %s", core().getCssValue("top"), core().getCssValue("left"));
     }
 
+    @JDIAction("'{name}' has position from top")
+    public int getTopPosition() {
```

Comment:
topPosition() is good name, we have a minimum methods with get*

Reply:
Modified the names.